geopandas/io/arrow.py

```diff
@@ -860,12 +860,27 @@ def _read_feather(path, columns=None, **kwargs):
 
 
 def _get_parquet_bbox_filter(geo_metadata, bbox):
+    primary_column = geo_metadata["primary_column"]
 
-    if not _check_if_covering_in_geo_metadata(geo_metadata):
-        raise ValueError("No covering bbox in parquet file.")
+    if _check_if_covering_in_geo_metadata(geo_metadata):
+        bbox_column_name = _get_bbox_encoding_column_name(geo_metadata)
+        return _convert_bbox_to_parquet_filter(bbox, bbox_column_name)
 
-    bbox_column_name = _get_bbox_encoding_column_name(geo_metadata)
-    return _convert_bbox_to_parquet_filter(bbox, bbox_column_name)
+    elif geo_metadata["columns"][primary_column]["encoding"] == "point":
+        import pyarrow.compute as pc
+
+        return (
+            (pc.field((primary_column, "x")) >= bbox[0])
+            & (pc.field((primary_column, "x")) <= bbox[2])
+            & (pc.field((primary_column, "y")) >= bbox[1])
+            & (pc.field((primary_column, "y")) <= bbox[3])
+        )
+
+    else:
+        raise ValueError(
+            "Specifying 'bbox' not supported for this Parquet file (it does not have "
+            "a bbox covering column, or does not use 'point' encoding)."
```

Comment:
I don't think this is correct English. Maybe `it does not have  a bbox covering column, nor does it use 'point' encoding`?